aten/src/ATen/native/quantized/cpu/qrelu.cpp

```diff
@@ -3,6 +3,7 @@
 #include <ATen/core/op_registration/op_registration.h>
 #include <ATen/native/TensorIterator.h>
 #include <ATen/native/cpu/Loops.h>
+#include <ATen/quantized/Quantizer.h>
```

Comment:
unused?

Reply:
I think I need it for the `at::quantize_val`.